src/transformers/models/patchtst/configuration_patchtst.py

```diff
@@ -0,0 +1,270 @@
+        ff_dropout (`float`, *optional*, defaults to 0.0):
+            The dropout probability used between the two layers of the feed-forward networks.
+        bias (`bool`, *optional*, defaults to `True`):
+            Consider bias in the feed-forward networks.
+        activation_function (`str`, *optional*, defaults to `"gelu"`):
+            The non-linear activation function (string) in the encoder.`"gelu"` and `"relu"` are supported.
+        pre_norm (`bool`, *optional*, defaults to `True`):
+            Normalization is applied before self-attention if pre_norm is set to True. Otherwise, normalization is
+            applied after residual block.
+        positional_encoding (`str`, *optional*, defaults to `"sincos"`):
+            Positional encodings. `"zeros"`, `"normal"`, `"uniform"' and `"sincos"` are supported.
+        learn_pe (`bool`, *optional*, defaults to `False`):
+            Whether the positional encoding is updated during training.
+        use_cls_token (`bool`, *optional*, defaults to `False`):
+            Whether cls token is used.
+        init_std (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated normal weight initialization distribution.
+        shared_projection (`bool`, *optional*, defaults to `True`):
+            Sharing the projection layer across different channels in the forecast head.
+        seed_number (`Optional`, *optional*):
+            Use seed number for random masking.
+        scaling (`Union`, *optional*, defaults to `"mean"`):
+            Whether to scale the input targets via "mean" scaler, "std" scaler or no scaler if `None`. If `True`, the
+            scaler is set to "mean".
+        mask_input (`bool`, *optional*, defaults to False):
+            Apply masking during the pretraining.
+        mask_type (`str`, *optional*, defaults to `"random"`):
+            Masking type. Only `"random"` is currently supported.
+        random_mask_ratio (`float`, *optional*, defaults to 0.5):
+            Masking ratio is applied to mask the input data during random pretraining.
+        forecast_mask_patches (`List`, *optional*, defaults to `[2, 3]`):
+            List of patch lengths to mask in the end of the data.
+        forecast_mask_ratios (`List`, *optional*, defaults to `[1, 1]`):
+            List of weights to use for each patch length. For Ex. if patch_lengths is [5,4] and mix_ratio is [1,1],
+            then equal weights to both patch lengths. Defaults to None.
+        channel_consistent_masking (`bool`, *optional*, defaults to `False`):
+            If channel consistent masking is True, all the channels will have the same masking.
+        unmasked_channel_indices (`list`, *optional*):
+            Channels are not masked during pretraining.
+        mask_value (`int`, *optional*, defaults to 0):
+            Mask value to set.
+        pooling (`str`, *optional*, defaults to `"mean"`):
```

Comment:
To match with other configs
```suggestion
        pooling_type (`str`, *optional*, defaults to `"mean"`):
```